FWCore/Reflection/test/BuildFile.xml

```diff
@@ -6,3 +6,4 @@
 
 <test name="TestFWCoreReflectionCheckClassVersion" command="run_checkClassVersion.sh"/>
 <test name="TestFWCoreReflectionDumpClassVersion" command="run_dumpClassVersion.sh"/>
+<test name="TestFWCoreReflectionClassVersionUpdate" command="run_classVersionUpdate.sh"/>
```

Comment:
@smuzaffar Another question on test dependencies. Is it possible for a test to declare a dependence on a library that is defined in the same `BuildFile.xml` (i.e. `FWCoreReflectionTestObjects` in this case)?

Reply:
@makortel , yes by adding `<lib name="FWCoreReflectionTestObjects"/>` in the <test .... > </test>` block

Reply:
ah wait, this works for `<bin .../>` but let me check if it works for `<test...../>` too

Reply:
No, currently `<test ..../>`  does not  treat `<lib name="SomeLocalLib"/>` properly but it can be easily fixed.

Reply:
Thanks!